PowerPointLabs/PowerPointLabs/Views/ProcessingStatusForm.cs

```diff
@@ -1,39 +0,0 @@
-﻿using System.Windows.Forms;
-
-namespace PowerPointLabs.Views
-{

```

Comment:
The original ProcessingStatusForm is written in WinForm. Since eLearningLab also needs to display ProgressingStatusBar to show progress, so I rewrote `ProcessingStatusForm` in WPF and deleted the original.

Reply:
Oh cool, thus it affects this too, I see.